### ISSUE TITLE
Product category’s do not work with variations

### ISSUE THREAD

**Issue by Nitrozik** [somewherewarm/woocommerce-product-dependencies#27]
Aloha!

We have one product in our store which is an add-on for another product, so we set it up so it’s only available if you buy a product from category A. In product category A we have products with product variations. If you select a product variation out of Category A and add it to your cart, the plugin does not recognize it as a product of Category A so you cannot add the add-on product to your cart. There is no way of adding a product variation to a category as far as i know. Could you check it out and see if there is a fix for this?

Thank you so much

**Comment by franticpsyx** [comment]
Thanks for reporting this @Nitrozik -- we'll do our best to have a look at this in the next few days :)

**Comment by franticpsyx** [comment]
@jimjasson can you replicate this?

**Comment by jimjasson** [comment]
Confirmed.

Here's what I did:

- I created a 'Hoodies' category with 'Hoodie' (Simple Product) and 'Hoodie with Zipper' (Variable Product).

- I created a Simple Product, 'Polo', and added a Purchase Dependency on the 'Hoodies' category I previously created.

- I added the 'Hoodie with Zipper', choosing the 'Large Size' variation, to the Cart.

- I tried to add 'Polo' to the Cart but the error notice came up requesting to add a product from the 'Hoodies' category.

- I added 'Hoodie' to the Cart.

- I re-tried adding 'Polo' to the Cart successfully.
